skbio/stats/composition.py

```diff
@@ -196,6 +202,9 @@ def multiplicative_replacement(mat, delta=None):
         delta = (1. / num_feats)**2
 
     zcnts = 1 - tot * delta
+    if np.any(zcnts) < 0:
+        raise ValueError('The multiplicative replacment created negative '
```

Comment:
Personally, I'm leaning towards keeping it a ValueError, since this method shouldn't return negative proportions.